src/hyperfine/export/markdown.rs

```diff
@@ -1,39 +1,157 @@
 use super::Exporter;
+
+use hyperfine::format::{Unit, format_duration_value};
 use hyperfine::types::BenchmarkResult;
 
 use std::io::Result;
 
-const MULTIPLIER: f64 = 1e3;
+macro_rules! TABLE_HEADER {
```

Comment:
I had tried and that ran into some issues, I think because I was trying to use `str`, but using `String` instead it's happy.

Converted to function.